### PR TITLE
Make server_tokens configurable per virtual-host

### LINKED ISSUE #1094: Hide Nginx Version
Add possibility to hide Nginx Server Version

### PR DESCRIPTION
**Pull request by alekna**
Makes `server_tokens` configurable per virtual-host without need to build custom images or mount custom configuration.

Sample docker-compose:
```
version: '2'

services:
  nginx-proxy:
    image: nginxproxy/nginx-proxy
    ports:
      - "80:80"
    volumes:
      - /var/run/docker.sock:/tmp/docker.sock:ro

  whoami:
    image: jwilder/whoami
    expose:
      - "8000"
    environment:
      - VIRTUAL_HOST=whoami.local
      - VIRTUAL_PORT=8000
      - SERVER_TOKENS=off
```

Fixes #1094 #1418

### CODE REVIEW

**Comment by alekna** [comment]
It's quite commonly requested feature and something I'm also looking to have as a built-in feature in `nginx-proxy`. @buchdag what are the odds of this change being accepted? Is there anything you would like me to improve in my PR?

**Comment by buchdag** [comment]
@alekna I made a review on this PR with two very minor changes, as soon as those changes are made I'll merge this.

**Comment by alekna** [comment]
I don't see any changes requested.

**Review by buchdag — CHANGES_REQUESTED** [review]

**Comment by buchdag** [comment]
My bad, I did not submit it.

File: test/test_headers/test_http.yml
Comment (alekna) [reply test/test_headers/test_http.yml:9]: done

File: test/test_headers/test_https.yml
Comment (alekna) [reply test/test_headers/test_https.yml:9]: done

**Review by buchdag — APPROVED** [review]
LGTM

**Pull request merged**
